services/efs-guardian/Dockerfile

```diff
@@ -205,7 +203,7 @@ ENV SC_BUILD_TARGET=development \
 
 WORKDIR /devel
 
-RUN chown -R efs:efs "${VIRTUAL_ENV}"
+RUN chown -R root:root "${VIRTUAL_ENV}"
```

Comment:
I undrstand this is only a development feature, i.e. it will not be accessible in production to users.
what is your plan to solve this? Is there an gh issue for it? if so add a note here.

Reply:
Depends on how fast I develop :) but it can happen that in the next release this might be still there.